langlib/langlib-test/src/test/resources/test-src/decimallib_test.bal

```diff
@@ -319,6 +319,165 @@ function testFromStringFunctionWithInvalidValues() {
     }
 }
 
+function testQuantize() {
```

Comment:
Let's add some tests with method call expressions and named arguments.

Reply:
Added.